proto/sentry_protos/snuba/v1/endpoint_trace_item_table.proto

```diff
@@ -39,4 +40,6 @@ message TraceItemColumnValues {
 message TraceItemTableResponse {
   repeated TraceItemColumnValues column_values = 1;
   PageToken page_token = 2;
+  optional QueryStats query_stats = 3;
+  optional QueryMetadata query_metadata = 4;
```

Comment:
I would put both of these objects inside another object, so that you only have to add one field to the response instead of two.

Reply:
these should also be repeated because there could be many queries in one request